ic-os/setupos/config/config.ini

```diff
@@ -1,10 +1,19 @@
 # Network Configuration File
 #
-# Update these settings according to your specific network requirements.
+# Update these settings according to your specific network and node requirements.
 
+# ------------------------------
+# Node Remuneration Settings (required)
+# ------------------------------
+# Update node_remuneration_type as per your node configuration.
+# Node remunerations should be of the form:
+# node_remuneration_type=typeX or node_remuneration_type=typeX.Y
+# Example:
+# node_remuneration_type=type3.1
+node_remuneration_type=
```

Comment:
@sasa-tomic is node_remuneration_type the proper name?

Reply:
The [registry field](https://github.com/dfinity/ic/pull/1608/files#diff-ef38f6c01be61d928aa7550dc45b2c0b8b0bb900b4dc967966ecb9450b23b173R190) is `node_reward_type`, so would be reasonable from my point of view to use the same name here.
Thanks for preparing this!